save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/TestSuitesDisplayer.kt

```diff
@@ -54,28 +54,38 @@ fun ChildrenBuilder.showAvaliableTestSuites(
                         +(testSuite.language ?: "")
                     }
                 }
-                p {
-                    +(testSuite.description ?: "")
+                div {
+                    className = ClassName("clearfix mb-1")
+                    div {
+                        className = ClassName("float-left")
+                        p {
+                            +(testSuite.description ?: "")
+                        }
+                    }
+                    div {
+                        className = ClassName("float-right")
+                        if (displayMode.shouldDisplayVersion()) {
+                            small {
+                                asDynamic()["data-toggle"] = "tooltip"
+                                asDynamic()["data-placement"] = "bottom"
+                                title = "Hash of commit with current test suite"
```

Comment:
need to update title, it's not a hash commit in most cases

Reply:
Ok, will update it

Reply:
Done